libs/@guardian/source-react-components/src/choice-card/ChoiceCard.stories.tsx

```diff
@@ -35,7 +37,38 @@ export default {
 };
 
 const Template: Story<ChoiceCardProps> = (args: ChoiceCardProps) => (
-	<ChoiceCard {...args} />
+	<div style={{ columns: 5 }}>
```

Comment:
Are these better as 3 separate stories?

Reply:
Moved this to a new story. What do you think?

Reply:
Actually just deleted it

Reply:
Do you think 2 stories, One for recommended theming and one for backwards compatibility. Or just one for recommended

Reply:
![Uploading Screenshot 2024-01-11 at 16.43.29.png…]()

Reply:
<img width="1324" alt="Screenshot 2024-01-11 at 16 43 29" src="https://github.com/guardian/csnx/assets/40648315/efc01bc1-9124-40c2-b5f3-fdd8cb5021b3">

Reply:
https://635a6fffacd30d393597c1ff-uixqjbdlzi.chromatic.com/?path=/docs/choicecard-theming--docs

Reply:
Sorry Ollie, actually thinking about it this might get a bit much if we do this for each component. Can we just have an additional story within the ChoiceCard stories that demonstrates an alternative theme?

Reply:
:'(

Reply:
https://635a6fffacd30d393597c1ff-qtrtddfvkq.chromatic.com/?path=/docs/choicecard--docs